### ISSUE TITLE
NameError for missing `os` import in `graphein.protein.graphs` 

### ISSUE THREAD

**Issue by kamurani** [a-r-j/graphein#296]
When using the `construct_graph` function in a jupyter notebook, I encountered this error: 

```
File ~/env/python3.10/site-packages/graphein/protein/graphs.py:764, in construct_graph(config, name, pdb_path, uniprot_id, pdb_code, df, chain_selection, model_index, df_processing_funcs, edge_construction_funcs, edge_annotation_funcs, node_annotation_funcs, graph_annotation_funcs, verbose)
    761 if pdb_path is not None and isinstance(pdb_path, Path):
    762     pdb_path = os.fsdecode(pdb_path)
...
--> 764 # If no config is provided, use default
    765 if config is None:
    766     config = ProteinGraphConfig()

NameError: name 'os' is not defined
```

In the `graphs.py` file, there is no `import os`.  Adding this fixes the error.  Not sure what's going on as I've never encountered this before?

**Comment by a-r-j** [comment]
Thanks for the bug report. No idea how this slipped through testing. I'll make a new release ASAP.

**Comment by a-r-j** [comment]
Should be resolved in 1.7.0 (`pip install graphein==1.7.0`)
